Projects/My_website/myWebsite/blog/models.py

```diff
@@ -26,7 +20,7 @@ class Post(models.Model):
     date = models.DateField(auto_now=True)
     slug = models.SlugField(unique = True)
     content = models.TextField()
-    author = models.ForeignKey(Author,null=True,on_delete = models.SET_NULL,related_name = "posts")
+    author = models.ForeignKey(User,null=True,on_delete = models.SET_NULL,related_name = "posts")
     tags = models.ManyToManyField(Tag)
```

Comment:
@codal-uchoudhary Please add format your code. Seems like the code is formatted properly.